Sprint-3/quote-generator/quotes.js

```diff
@@ -491,3 +491,14 @@ const quotes = [
 ];
 
 // call pickFromArray with the quotes array to check you get a random quote
+function displayRandomQuote() {
+  const randomQuote = pickFromArray(quotes); // Select a random quote
+  document.getElementById("quote").innerText = `"${randomQuote.quote}"`; // Display the quote text
+  document.getElementById("author").innerText = `— ${randomQuote.author}`; // Display the author name
+}
+
+document
+  .getElementById("new-quote")
+  .addEventListener("click", displayRandomQuote);
+
+window.onload = displayRandomQuote;
```

Comment:
As a rule of thumb, always define your dom elements in the beginning of your code block
and please define them  something similar to this. 
`const quoateText= document.getElementById("quote")`
and then use these variables in your functions.
This is essential  to  write more clean codes.